modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -208,6 +208,18 @@ def build_partition(cls, partition_ids, column_widths):
         np.ndarray
             array with shape equals to the shape of `partition_ids` and
             filed with partition objects.
+
+        Notes
+        -----
+        partition_ids contains three levels of nesting. The first level contains lists
+        of files to read with corresponding columns as such:
+        partition_ids[i] -> [read(files, cols[0..n]), read(files, cols[n..m]), ...].
```

Comment:
Why include a description of the top level? I think the part about "The second level contains a list of object references for each read call:" is sufficient. It shouldn't be part of the API that `partition_ids[i]` comes from `read`.

Reply:
Fair enough